src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -596,6 +597,10 @@ static String getMainClassFromJar(String jarname) {
             if (opens != null) {
                 addExportsOrOpens(opens, true);
             }
+            String enablesNativeAccess = mainAttrs.getValue(ENABLE_NATIVE_ACCESS);
+            if ("true".equals(enablesNativeAccess)) {
```

Comment:
What I see done elsewhere is something like this:
```
isMultiRelease = Boolean.parseBoolean(
                            attr.getValue(Attributes.Name.MULTI_RELEASE));
```

Reply:
Ah, can do. I thought that it would throw an exception if the input wasn't `true` or `false`, just like e.g. [`Integer::parseInt`](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/lang/Integer.html#parseInt(java.lang.String)) TIL.